resources/scripts/generate-build-data.sh

```diff
@@ -293,6 +314,11 @@ function normaliseTestsWithoutStacktrace() {
     jqEdit 'map(del(.errorStackTrace))' "${file}"
 }
 
+function normaliseCoberturaSummary() {
+    file=$1
+    jqEdit '.results.elements | reduce to_entries[] as $o ({}; .[$o.value.name] += { ration: $o.value.ratio, numerator: $o.value.numerator, denominator: $o.value.denominator})' "${file}"
```

Comment:
Given https://github.com/elastic/apm-pipeline-library/pull/1214/files#diff-a69ef83a90d59301869896bf58d9aad9e82b9ed7eec9dfa7b20ce28336c4856d

it produces
src/test/resources/tests-cobertura.json